tests/test_eclendpointscaling.cpp

```diff
@@ -111,9 +110,21 @@ BOOST_AUTO_TEST_CASE (NoScaling)
 
     const auto eps = SF::TwoPointScaling{ smin, smax };
 
-    const auto s_eff = eps.eval(tep, sp);
+    // Input saturation -> Scaled saturation
+    {
+        const auto sp    = associate(s);
+        const auto s_eff = eps.eval(tep, sp);
+
+        check_is_close(s_eff, expect);
+    }
 
-    check_is_close(s_eff, expect);
+    // Tabulated saturation -> Input saturation
+    {
+        const auto sp    = associate(expect);
+        const auto s_inp = eps.eval(tep, sp);
```

Comment:
> Should not this call `reverse()` instead?

You're right--that's a mistake.  Fixed in 258a58f.  The practical outcome of the test doesn't change in this case because the input and scaled saturation intervals are the same (and we're using the two-point scaling method), but we must of course use the correct function calls.